lib/LaTeXML/Package/pgfsys-latexml.def.ltxml

```diff
@@ -78,33 +81,53 @@ DefConstructor('\lxSVG@insertpicture{}',
   #   . '</ltx:picture>',
   sub {
     my ($document, $content, %props) = @_;
-    $document->openElement('ltx:picture');
-    $document->openElement('svg:svg',
-      version  => "1.1",
-      width    => $props{pxwidth}, height => $props{pxheight},
-      viewBox  => "$props{minx} $props{miny} $props{pxwidth} $props{pxheight}",
-      overflow => "visible");
-    $document->openElement('svg:g',
-      transform   => "matrix(1 0 0 -1 0 $props{flipnmove})",
-      _scopebegin => 1);
-    $document->absorb($content);
+    my $current = $document->getNode();
+    if (0 && $document->getNodeQName($current) =~ /^svg:/) {    # Already in svg:svg ?
```

Comment:
Is `0 &&` some new kind of enhanced commenting technique I'm not aware of?

Reply:
ha, you force-removed it just as I commented. Nevermind :>

Reply:
exactly!  But it's been uncommented (& fixed) now anyway